manifests/global.yaml

```diff
@@ -41,6 +41,9 @@ global:
   # Default tag for Istio images.
   tag: latest
 
+  # clusterID for the cluster
```

Comment:
My standard comment: this should be in ProxyConfig or MeshConfig (or in values.yaml of the chart that uses it).

It doesn't make any sense as a 'global' - in fact nothing in this file is global since we support multiple revisions, and each may have a different config.